lib/byron/bench/Restore.hs

```diff
@@ -183,12 +183,16 @@ main = do
     configs <- getEnv "BYRON_CONFIGS"
     nodeDB <- getEnv "NODE_DB"
 
+    -- Temporary directory for storing socket
+    tmpDir <- getCanonicalTemporaryDirectory
+        >>= \tmpRoot -> createTempDirectory tmpRoot "cw-byron"
+
     let genesisHash =
             B8.unpack
             . hex
             . getGenesisBlockHash
             $ blockchainParameters @'Mainnet
-    let socketPath = "cardano-node.socket"
+    let socketPath = tmpDir </> "cardano-node.socket"
     let args =
             [ "run"
```

Comment:
Out of curiosity, where are the configs below coming from exactly? Are they also generated and written somewhere to disk ?

Reply:
They are coming from https://github.com/input-output-hk/cardano-wallet/blob/65eafca208ff8c2c544e4a094f4fe3cf9b77c3b4/nix/haskell.nix#L94